.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## Description
+*Briefly describe the changes introduced by this PR*
+
+
+## What does this PR do?
+- [ ] Bugfix
+- [ ] New feature
+- [ ] Refactor
+- [ ] Documentation
+
+## What are the relevant issues?
+- [ ] Jira issue [JIRA]()
+
+---
+
+## Checklist
+- [ ] I have updated the documentation accordingly
+- [ ] I have added tests to cover my changes
+- [ ] All new and existing tests passed
+- [ ] My code follows the code style of this project
+  - [Git Style Guide](https://refaktor.atlassian.net/wiki/spaces/GymTrackr/pages/25264131/Git+Guidelines)
+  - [Code Review Guidelines](https://refaktor.atlassian.net/wiki/spaces/GymTrackr/pages/25067538/Code+Review+Guidelines)
```

Comment:
@ManuSG8 
Please look at these two documents when you have time, we have to start thinking about maintaining a solid pattern in terms of commit messages, commit history, pull request structure, documentation, etc...